src/common/api-config/api.config.service.ts

```diff
@@ -587,4 +587,17 @@ export class ApiConfigService {
   getNftProcessMaxRetries(): number {
     return this.configService.get<number>('nftProcess.maxRetries') ?? 3;
   }
+
+  getMicroServiceUrl(): string | undefined {
```

Comment:
getMicroServiceUrl is too generic.
maybe a better name would be `getMaiarExchangeUrl`